test/pipetool.uts

```diff
@@ -15,8 +15,6 @@ s > d1 > c
 d1 > tf > t
 
 p = PipeEngine(s)
-p.graph(type="png", target="test.png")
```

Comment:
Is there a good reason to remove these lines?

Reply:
They were silently crashing, but triggered coverage. @p-l- Chose to remove the silently crashing tests. We can implement it again with mocking in the future